my-app/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make `my-app` the root-folder of your app. I noticed that your `my-app` react app is nested within another folder called `MATH-MAGICIEN` and that folder contains your README file. You only need one README file for your app.